From e30dd2215136492ccf887d585bef7d24fbe579d7 Mon Sep 17 00:00:00 2001
From: Steven Laver <lavers@google.com>
Date: Tue, 17 Mar 2020 14:10:58 -0700
Subject: [PATCH 10/12] bluetooth: update SavedBluetoothTwsDeviceUpdater for
 superclass ctor signature change

In 130629b1dbe3f43fe7695c606224b805ec413e7b, the constructor signature
of BluetoothDeviceUpdater changed; a Context parameter was added. Update
its subclass, SavedBluetoothTwsDeviceUpdater, accordingly.

Change-Id: I07945f96e0ff6b1e82b1c9f5732eff69cecdca97
---
 .../settings/bluetooth/SavedBluetoothTwsDeviceUpdater.java    | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/com/android/settings/bluetooth/SavedBluetoothTwsDeviceUpdater.java b/src/com/android/settings/bluetooth/SavedBluetoothTwsDeviceUpdater.java
index d9c3d217b4..f4a5638c8b 100644
--- a/src/com/android/settings/bluetooth/SavedBluetoothTwsDeviceUpdater.java
+++ b/src/com/android/settings/bluetooth/SavedBluetoothTwsDeviceUpdater.java
@@ -70,10 +70,10 @@ public class SavedBluetoothTwsDeviceUpdater extends BluetoothDeviceUpdater {
         super(context, fragment, devicePreferenceCallback);
     }
 
-    SavedBluetoothTwsDeviceUpdater(DashboardFragment fragment,
+    SavedBluetoothTwsDeviceUpdater(Context context, DashboardFragment fragment,
             DevicePreferenceCallback devicePreferenceCallback,
             LocalBluetoothManager localBluetoothManager) {
-        super(fragment, devicePreferenceCallback, localBluetoothManager);
+        super(context, fragment, devicePreferenceCallback, localBluetoothManager);
     }
 
     @Override
-- 
2.28.0

